diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -372,7 +372,7 @@ namespace diff_drive_controller
     setOdomPubFields(root_nh, controller_nh);
 
     // Set dynamic reconfigure params defaults to the static params provided:
-    DiffDriveControllerConfig config;
+    DiffDriveControllerConfig config(config_default_);
```

Comment:
Well, we should actually keep this commit too. It's better to have the defaults, than random bits from memory.

Looking in the autogenerated header for the cfg, I can see there's no constructor and copy constructor. So it's using the default ones. That explains the constructor itself doesn't initialize any attribute of the class. :sob: